<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LOGIN</class>
 <widget class="QMainWindow" name="LOGIN">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1925</width>
    <height>1033</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Inventory and Sales Management System</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: white;</string>
   </property>
   <widget class="QLabel" name="PIC">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1921</width>
      <height>982</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/JJimages/PIC_log_sign_in.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>130</y>
      <width>451</width>
      <height>331</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/JJimages/JJLOGO.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>0</y>
      <width>931</width>
      <height>982</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QCheckBox" name="checkBox_rememberme">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>610</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #022162;
font-family: &quot;Verdana&quot;, sans-serif; 
background: transparent;</string>
     </property>
     <property name="text">
      <string>Remember me</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_xtolanding">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>50</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>-1</pointsize>
       <weight>87</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 13px; 
padding: 1px; 
font-weight: 700;
font-family: &quot;Verdana&quot;, sans-serif; 
font-size: 18px;
background-color: #374550; 
color: white;</string>
     </property>
     <property name="text">
      <string>‚Üê Back to Landing</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>130</y>
       <width>261</width>
       <height>131</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>47</pointsize>
       <weight>87</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #022162;
font-weight: 700;
font-family: &quot;Verdana&quot;, sans-serif; 
background: transparent;</string>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QLabel" name="username">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>310</y>
       <width>181</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>18</pointsize>
       <weight>87</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #022162;
font-weight: 700;
font-family: &quot;Verdana&quot;, sans-serif; 
background: transparent;</string>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QLabel" name="password">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>460</y>
       <width>181</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>18</pointsize>
       <weight>87</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #022162;
font-weight: 700;
font-family: &quot;Verdana&quot;, sans-serif; 
background: transparent;</string>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="login_usrname">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>370</y>
       <width>841</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid #022162; 
border-radius: 25px; 
padding: 5px;
font-family: &quot;Verdana&quot;, sans-serif; </string>
     </property>
     <property name="placeholderText">
      <string>  Enter your username</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="login_password">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>530</y>
       <width>831</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid #022162; 
border-radius: 25px; 
padding: 5px;
font-family: &quot;Verdana&quot;, sans-serif; </string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>  Enter your password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_LOGIN">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>700</y>
       <width>241</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>15</pointsize>
       <weight>87</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid #374550; 
border-radius: 25px; 
padding: 5px; 
font-weight: 700;
font-family: &quot;Verdana&quot;, sans-serif; 
background-color: #374550; 
color: white;</string>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QLabel" name="forgotPassLabel">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>680</y>
       <width>351</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>-1</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #022162;
font-family: &quot;Verdana&quot;, sans-serif; 
font-size: 14px;
background: transparent;</string>
     </property>
     <property name="text">
      <string>Forgot password? Only available for Shop Owners</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_forgotPass_login_page">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>720</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>13</pointsize>
       <weight>87</weight>
       <bold>true</bold>
       <underline>true</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 25px; 
padding: 5px; 
font-weight: 700;
font-family: &quot;Verdana&quot;, sans-serif; 
background-color: transparent; 
color: #022162;</string>
     </property>
     <property name="text">
      <string>Reset Password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="togglePasswordButton">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>540</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#togglePasswordButton {
    border: none;          /* Removes the border */
    background-color: transparent; /* Transparent background */
    padding: 2px;          /* Optional: Adds a bit of spacing */
}

</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../../Downloads/EYE.png</normaloff>../../../Downloads/EYE.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>45</width>
       <height>45</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_clearLogins">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>600</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 25px; 
padding: 5px; 
font-size: 20px;
font-family: &quot;Verdana&quot;, sans-serif; 
background-color: #c25b55; 
color: white;</string>
     </property>
     <property name="text">
      <string>Clear </string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="jframe">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>460</y>
      <width>951</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>771</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>24</pointsize>
       <weight>87</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #ffffff;
font-weight: 700;
font-family: &quot;Verdana&quot;, sans-serif; 
background: transparent;
</string>
     </property>
     <property name="text">
      <string>J &amp; J Roofsteel and Gutter Supply</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>110</y>
       <width>291</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>18</pointsize>
       <weight>87</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #ffffff;
font-weight: 700;
font-family: &quot;Verdana&quot;, sans-serif; 
background: transparent;
</string>
     </property>
     <property name="text">
      <string>Moalboal Branch</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>250</y>
       <width>631</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>16</pointsize>
       <weight>87</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #fff2bd; 
font-weight: 700;
font-family: &quot;Verdana&quot;, sans-serif; 
background: transparent; 
</string>
     </property>
     <property name="text">
      <string>Supply       |         Install       |         Repair    </string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1925</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../../../Downloads/jj_owner_resources.qrc"/>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
